user_test.go

```diff
@@ -59,9 +59,43 @@ func TestClient_UpdateUsers(t *testing.T) {
 
 	user := randomUser()
 
-	resp, err := c.UpdateUsers(user)
-	mustNoError(t, err, "update users")
-
-	assert.NotEmpty(t, resp[user.ID].CreatedAt)
-	assert.NotEmpty(t, resp[user.ID].UpdatedAt)
+	t.Run("update users", func(t *testing.T) {
+		resp, err := c.UpdateUsers(user)
+		mustNoError(t, err, "update users")
+
+		assert.Contains(t, resp, user.ID)
+		assert.NotEmpty(t, resp[user.ID].CreatedAt)
+		assert.NotEmpty(t, resp[user.ID].UpdatedAt)
+	})
+
+	t.Run("partial update", func(t *testing.T) {
```

Comment:
Run is not suitable for stacking tests. That is, this run depends on the previous.